app/templates/_pom.xml

```diff
@@ -172,6 +172,12 @@
             <groupId>io.springfox</groupId>
             <artifactId>springfox-swagger2</artifactId>
             <version>${springfox.version}</version>
+            <exclusions>
```

Comment:
This could be discussed in a separate thread, but the MapStruct version included in jhipster_2.17.0 (jdk == 8 1.0.0.CR1) is conflicting with the one that the included version of springfox-swagger2 includes (jdk <=7 1.0.0.Beta3), resulting in conflicts by having both versions one for jdk8 and the other for jdk<8